python/pyspark/sql/session.py

```diff
@@ -277,8 +277,10 @@ def getOrCreate(self) -> "SparkSession":
                     # Do not update `SparkConf` for existing `SparkContext`, as it's shared
                     # by all sessions.
                     session = SparkSession(sc, options=self._options)
-                for key, value in self._options.items():
-                    session._jsparkSession.sessionState().conf().setConfString(key, value)
+                else:
```

Comment:
Hm, in Scala side, `applyModifiableSettings` is always invoked when the default Spark session is not found, and `SparkContext` is already running. Is this matched with Scala side? Seems not.

Reply:
> 

Hmmm,  in Scala side, applyModifiableSettings is always invoked when the default Spark session is found?

Reply:
oops i misread